Source/Renderer/UniformState.js

```diff
@@ -935,6 +961,14 @@ define([
 
         this._frameState = frameState;
         this._temeToPseudoFixed = Transforms.computeTemeToPseudoFixedMatrix(frameState.time, this._temeToPseudoFixed);
+
+        var fov = camera.frustum.fov;
+        var viewport = this._viewport;
+        if (viewport.width > viewport.height * camera.frustum.aspectRatio) {
```

Comment:
Are you sure `mainViewport[3]` is `width`?  In Cesium, the array would be `[x, y, width, height]`.

Are you sure `pixelAspect` ratio is `width / height` like `aspectRatio`?

Perhaps ask Greg to double check these.

Right now, I think the `if` statements reads like "if (width > (height * (width / height))", which is "if (width > width)"